README.md

```diff
@@ -227,6 +227,10 @@ Scenario Outline: Fetching <key> API endpoint from root endpoint
     | public_gists_url |
 ```
 
+Extracting a value from the response json body is set by default. So the syntax: `pick response json <key> as <key>` is the same with 
+the syntax `pick response json body <key> as <key>`. If you want to test some other values from your header response, 
+you need to specify your command by adding a word `header`: `pick response json header <key> as <key>`
```

Comment:
It can be reworeded imo, I'll thing of something.

Reply:
Assuming we change the matcher as said in my other comment, I suggest we remove this block and instead add a header in the example.

Reply:
Yes, I deleted it and add an example for this case.